pyne/compare_data.py

```diff
@@ -0,0 +1,37 @@
+"""this method compare two columns of results and errors, return
+   ture if the two columns have the same results otherwise return
+   false. Also return the list of absolute difference btw these 
+   two columns"""
+def compare(mode,file1,file2):
```

Comment:
I think this function should NOT try to compare files.

It should only compare numbers.

`def compare(val1,err1, val2, err2, mode)`